app/lib/registration/username_onboarding_screen.dart

```diff
@@ -190,3 +160,105 @@ class UsernameOnboardingScreen extends HookWidget {
     return handle.validationError();
   }
 }
+
+class _AddButton extends StatelessWidget {
+  const _AddButton({required this.isSubmitting, required this.onPressed});
+
+  final bool isSubmitting;
+  final VoidCallback onPressed;
+
+  @override
+  Widget build(BuildContext context) {
+    final colors = CustomColorScheme.of(context);
+    final loc = AppLocalizations.of(context);
+    return Container(
+      padding: const EdgeInsets.symmetric(horizontal: Spacings.m),
+      width: isSmallScreen(context) ? double.infinity : null,
+      child: OutlinedButton(
+        style: OutlinedButton.styleFrom(
+          textStyle: customTextScheme.labelMedium,
+          backgroundColor: colors.accent.primary,
+          foregroundColor: colors.function.toggleWhite,
+        ),
+        onPressed: isSubmitting ? null : onPressed,
+        child:
+            isSubmitting
+                ? SizedBox(
+                  height: 20,
+                  width: 20,
+                  child: CircularProgressIndicator(
+                    strokeWidth: 2,
+                    valueColor: AlwaysStoppedAnimation<Color>(
+                      colors.text.primary,
```

Comment:
Most likely this should `colors.backgroundBase.secondary`. Otherwise, it is almost black on a black button.